index.html

```diff
@@ -1 +1,59 @@
-this is the master file. note: do not write on the master file. create a branch.
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Unique Computers</title>
+</head>
+<body>
+   <div id="container">
+       <section>
+           <div id="homebanner">
+               <img src="" alt="">
```

Comment:
Give the sections unique ID and also remove the img tag, we will add the image directly with css.